cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -28,7 +28,7 @@
                         block__flush-sides">
                 {{ featured_content.render(block.value) }}
             </div>
-        {% elif block.block_type == 'feedback' %}
+        {% elif block.block_type == 'feedback' or block.block_type == 'referred_feedback' %}
```

Comment:
If we could somehow use class inheritance to unify these feedback blocks, then this could be made simpler and we wouldn't have to enumerate each block type here.